examples/projects/javascript-vanilla-with-webpack/README.md

```diff
@@ -0,0 +1,30 @@
+# Integrate `bpmn-visualization` in a vanilla webpack JavaScript project
+
+## Usage
+
+To run locally:
+
+1. `npm install`
+2. `npm run serve`
+3. [localhost app](http://localhost:8080)
+
+You should see a BPMN diagram displayed in the page
+
+![BPMN diagram in the home page](docs/home.png)
```

Comment:
Is it the end of the previous sentence ?

Reply:
Not really, I am going to rewrite it 😸

Reply:
Done in f5d24f6